fe/fe-core/src/main/java/org/apache/doris/nereids/properties/PhysicalProperties.java

```diff
@@ -106,6 +106,10 @@ public DistributionSpec getDistributionSpec() {
         return distributionSpec;
     }
 
+    public boolean isDistributorProperties() {
```

Comment:
maybe `distributionOnlyProperties` is a better name

Reply:
done